File: LinkedList.py
class Node:
    def __init__(self, data):
        self.data = data
        self.next = None
        
class LinkedList:
    def __init__(self):
        self.head = None
        
    def append(self,data):
        nn = Node(data)
        if self.head is None:
            self.head = nn
            return 
        
        temp = self.head
        while temp.next:
            temp = temp.next
            
        temp.next = nn;
        
    def display(self):
        temp = self.head
        
        while temp:
            print(temp.data,end=" ");
            temp = temp.next
            
l1 = LinkedList();
l1.append(10);
l1.append(20);
l1.append(30);
l1.append(40);
l1.display();

File: Linkedlist.java
class Node{
    int data;
    Node next;
    
    Node(int data){
        this.data = data;
        this.next = null;
    }
}


class Main{
    
    Node head;
    
    void append(int data)
    {
        Node nn = new Node(data);
        if(head == null) 
        {
            head = nn;
            return;
        }
        Node temp = head;
        while(temp.next != null) temp = temp.next;
        temp.next = nn;
    }
    
    void display()
    {
        Node temp = head;
        while(temp != null)
        {
            System.out.print(temp.data+" ");
            temp = temp.next;
        }
        System.out.println();
    }
    
    public static void main(String args[])
    {
        Main list = new Main();
        Main list2 = new Main();
        list2.append(100);
        list.append(10);
        list.append(20);
        list.append(30);
        list.append(40);
        list.display();
        list2.display();
    }
}
